modules/builds-secrets-overview.adoc

```diff
@@ -66,7 +66,7 @@ When you modify the value of a secret, the value used by an already running pod
 
 Updating a secret follows the same workflow as deploying a new container image. You can use the `kubectl rolling-update` command.
 
-The `resourceVersion` value in a secret is not specified when it is referenced. Therefore, if a secret is updated at the same time as pods are starting, then the version of the secret is used for the pod is not defined.
+The `resourceVersion` value in a secret is not specified when it is referenced. Therefore, if a secret is updated at the same time as pods are starting, then the version of the secret that is used for the pod is not defined.
```

Comment:
According to ISG, we should avoid if...then
>! if...then conj
Avoid using “then” to introduce an independent clause that follows an “if” clause. For example, write “If
you set a password, access is restricted,” not “If you set a password, then access is restricted.”